server_admin/topics/threat/compromised-tokens.adoc

```diff
@@ -6,6 +6,8 @@ There are a few things you can do to mitigate access tokens and refresh tokens f
 Another thing you can do to mitigate leaked access tokens is to shorten their lifespans.  You can specify this within the <<_timeouts, timeouts page>>.
 Short lifespans (minutes) for access tokens for clients and applications to refresh their access tokens after a short amount of time. If an admin detects a leak, they can logout all user sessions to invalidate these refresh tokens or set up a revocation policy. Making sure refresh tokens always stay private to the client and are never transmitted ever is very important as well.
 
+Other way of mitigating leaked access tokens and refresh tokens is to issue these tokens as Holder-of-Key token. <<_mtls-client-certificate-bound-tokens, OAuth 2.0 Mutual TLS Client Certificate Bound Access Token>> shows how to issue these tokens as Holder-of-Key token instead of bearer token.
+
```

Comment:
You can also mitigate against leaked access tokens and refresh tokens by issuing these tokens as holder-of-key tokens. See <<_mtls-client-certificate-bound-tokens, OAuth 2.0 Mutual TLS Client Certificate Bound Access Token>> to learn how.